centreon/packages/ui/src/StatusChip/index.tsx

```diff
@@ -40,8 +40,13 @@ const StatusChip = ({
     severityCode
   });
 
-  const lowerLabel = (name: string): string =>
-    name.charAt(0).toUpperCase() + name.slice(1).toLowerCase();
+  const lowerLabel = (name: string): string => {
+    if (equals(name, 'OK')) {
+      return name;
+    }
```

Comment:
I do think, you just have to change the label itself. Not create a very specific condition

Reply:
I'm not event sure that changing the case from within the Component is a good idead to begin with. Why don't we just handle the case from where the label prop is passed?

Reply:
We have discussed with @adr-mo in order to retrieve the formatted label from the API but it's not possible, **currently**, because RS API is not a API v2.
So yep, we have to handle it outside of the function but still inside the component